apps/designer/app/routes/rest/css.$projectId.$pageId.tsx

```diff
@@ -0,0 +1,46 @@
+import { ActionFunction, json } from "@remix-run/node";
+import env from "~/env.server";
+import { sentryException } from "~/shared/sentry";
+import { generateCssText } from "~/shared/css-utils";
+import { BuildParams } from "~/shared/router-utils";
+
+export const loader: ActionFunction = async ({ request, params }) => {
+  try {
+    const projectId = params.projectId ?? undefined;
+    const pageId = params.pageId ?? undefined;
+
+    if (projectId === undefined) {
+      throw json("Required project id", { status: 404 });
+    }
+
+    if (pageId === undefined) {
+      throw json("Required page id", { status: 404 });
+    }
```

Comment:
With these error messages, we should probably use `400`. Same with `Required project info` below